server/views/pages/overview/partials/overviewTabContents.njk

```diff
@@ -1,58 +1,60 @@
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
 
-    <div class="moj-page-header-actions">
-  
-      <div class="moj-page-header-actions__title">
-        <h2 class="govuk-heading-m">Goals in progress</h2>
-      </div>
+  {% if actionPlan.problemRetrievingData !== true %}
```

Comment:
This if statement is in the wrong place - it looks like you've wrapped the entire tab in it
You want the if statement only wrapping the for loop

Reply:
Possibly. I was thinking we don't want to show the "Add goal" button either, since our API is down.

Regardless, this will need changing when we can't retrieve functional skills etc, so I wasn't too concerned.

Any suggestions @chrimesdev, or shall we leave this as it is for now?

Reply:
On reflection I'm happy that we leave it for now - I suspect we'll be back doing stuff on this page soon (!!), so we can perhaps consider changing/moving it as and when,
I think its probably fine for now 👍